modules/deploy/pages/deployment-option/self-hosted/kubernetes/k-25.2-beta.adoc

```diff
@@ -0,0 +1,121 @@
+= Try the 25.2 Beta of the Redpanda Operator
+:page-beta: true
+:description: Deploy the 25.2 beta release of the Redpanda Operator. This version of the Redpanda Operator defaults to cluster scope so a single instance of the Operator can manage multiple Redpanda resources in different namespaces.
```

Comment:
"defaults" here feels misleading relative to [this PR](https://github.com/redpanda-data/docs/pull/1308) where you say support for multiple operators is removed. 

Maybe just "is cluster scoped"?
```suggestion
:description: Deploy the 25.2 beta release of the Redpanda Operator. This version of the Redpanda Operator is cluster scope so a single instance of the Operator can manage multiple Redpanda resources in different namespaces.
```